src/pods/canvas/canvas.business.ts

```diff
@@ -73,3 +79,75 @@ export const calculateRelationXCoordinate = (
   xOrigin: calculateRelationXCoordinateOrigin(tableOrigin, tableDestination),
   xDestination: calculateRelationXCoordinateEnd(tableOrigin, tableDestination),
 });
+
+const seekField = (
+  fieldId: GUID,
+  YPosition: number,
+  fields: FieldVm[]
+): seekResult => {
+  let found = false;
+  let parentCollapsed = false;
```

Comment:
Mmm... I think this is not correct, we can add a test for this, let's say we have:

field `countryId`

```
id
name
▶ address (**collapsed**)
    postcode
    ▼ country 
        countryId
        name
``` 

In this case address is collapsed and Y should stick to the address field (cannot reach countryId), since we are always initializing `parentCollapsed` to false in each `seekField` recursive call it could fail.

How to solve this issue:

- First create a test for this (ensure it fails, if it doesn't fail maybe what I'm saying makes no sense :D).
- Then implement the remediation for this and ensure all tests are green